practice-one/src/Components/Section/Header.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const HeaderCarousel = ({
+  childrenBlack,
+  childrenGray,
+}) => (
+  <h1 className="text-black text-6xl font-bold pt-[3rem] pb-24">
+    <span className="text-black">{childrenBlack}</span>
+    <br />
+    <span className="text-gray-500">{childrenGray}</span>
+  </h1>
+);
+
+HeaderCarousel.propTypes = {
+  childrenBlack: PropTypes.node.isRequired,
+  childrenGray: PropTypes.node.isRequired,
+};
+
+const HeaderTab = ({
+  children,
+  onClick,
+}) => (
+  <li className="list-none hover:rounded text-gray-500">
+    <button

```

Comment:
this one is a `a` tag

Reply:
Yes, I changed `button` tag into `a` tag and add new handel for hover `a` tag